pyccel/parser/syntactic.py

```diff
@@ -636,7 +636,6 @@ def _visit_FunctionDef(self, stmt):
             ls = decorators['types'].args
             while i<len(ls) :
                 arg = ls[i]
-
```

Comment:
Please try to avoid adding/removing lines in a file where no other changes are made. All it does is add an extra file for reviewers